www/js/mission-control.js

```diff
@@ -212,7 +215,7 @@ function loadDesign(name) {
 function acceptName() {

```

Comment:
I think all of these functions (saveDesign, refreshSavedBds, loadDesign, and acceptName) all moved to rover-api.js. Shouldn't they all be removed from this file?